Assets/02.Scripts/02-8. UI/DailyReport/DailyReportPresenter.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using UnityEngine;
+
+public class DailyReportPresenter : MonoBehaviour
+{
+    [SerializeField]
+    private DailyReportModel _dailyReportModel;
+    
+    private UI_DailyReport _dailyReport;
```

Comment:
```suggestion
    private UI_DailyReport _dailyReportView;
```
필드명 뒤에 View가 붙으면 좋을 것 같습니다.